docs/2.1. Behaviour - Server Side.md

```diff
@@ -12,13 +12,13 @@ The initial state of Media Profiles of a Sender MUST be an empty JSON array `[]`
 
 Non-empty array of Media Profiles MUST be treated as unordered unless `preference` property exists.
 
-Once an array of Media Profiles is accepted, a Sender MUST have a Flow and Source which meets at least one profile. This constraint MUST be adhered to when an external change (i.e. not from the `/media-profiles` endpoint) modifies any aspect of the stream. If the change does not satisfy its Media Profiles, the Sender MUST NOT transmit the signal over the network. The Sender is REQUIRED to take the following actions:
+Once `PUT /media-profiles` is requested, the Node MUST respond and the Sender MUST change the Flow and/or the Source to meet at least one profile as soon as possible. This constraint MUST be adhered to when an external change (i.e. not from the `/media-profiles` endpoint) modifies any aspect of the stream. If the change does not satisfy the Media Profiles, the Sender MUST NOT transmit the signal over the network. The Sender is REQUIRED to take the following actions:
```

Comment:
```suggestion
Once `PUT /media-profiles` is requested, the Node MUST validate the request and present it to the source before responding. The Sender MUST change the Flow and/or the Source to meet at least one profile as soon as possible. This constraint MUST be adhered to when an external change (i.e. not from the `/media-profiles` endpoint) modifies any aspect of the stream. If the change does not satisfy the Media Profiles, the Sender MUST NOT transmit the signal over the network. The Sender is REQUIRED to take the following actions:
```

Reply:
Validating should be mentioned, but what is "source"? What if the underlying implementation is software only?

Reply:
The IS-04 definition is a Source is the origin of the content... which I assume is what needs the EDID to change the resolution regardless of the implementation

Reply:
I may be wrong, but in my understanding [a logical Source](https://specs.amwa.tv/is-04/releases/v1.3.1/docs/5.0._Data_Model.html#sources-flows-and-grains) isn't required to represent an HDMI Source which needs the EDID. I don't think we can present something to "an abstract logical point", it's just a representation of something real which we should communicate to. After we communicated to it, the logical Source reflects changes.

Reply:
I removed the source part to avoid confusion